tests/system/lab/README.rst

```diff
@@ -0,0 +1,25 @@
+
+Script to automate FogLAMP Lab
+------------------------------
+
+1. Install git i.e. `sudo apt install git`
+
+2. Clone FogLAMP repo and `cd tests/system/lab/`
+
+3. Check and set the configuration in `test.config`
+
```

Comment:
4. Please add `ARM I2C interface enabled` & once that's done, reboot your RPi to let the changes propagate.
Because we are using `enviroPhat` plugin for lab.

Reply:
Done